planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -1323,6 +1328,50 @@ void ObstacleAvoidancePlanner::publishDebugDataInOptimization(
       debug_wall_markers_pub_->publish(debug_wall_marker);
       debug_data_.msg_stream << "      publishDebugVisualizationWallMarker:= "
                              << stop_watch_.toc("publishDebugVisualizationWallMarker") << " [ms]\n";
+
+      if (is_publishing_drivable_area_boundary_) {
```

Comment:
How about creating a function and calling it in `getDebugVisualizationMarker` func like the other debug markers instead of publishing drivable area seperately?